### DISCUSSION TITLE
Lab#2 Feedback

### DISCUSSION THREAD

**Discussion by iamyuchy** [Kingfisherzh/Yanjun-s-Lab-Hub#4] [General]
- Clear and useful button interactions
- For future implementation, maybe you can use the buttons as "Previous" and "Next" to show the tips for morning, afternoon, evening and midnight.

**Comment by zhonghaozhan** [comment]
The button interaction is straightforward; however, you might want to label the button on the default screen to let the users know what are their functions.
The midnight notification is a practical feature. When moving forward you could expand it.

**Comment by FadingWinds** [comment]
Maybe in addition to showing the text of "midnight" you can attach a little image of the moon. And you can also randomize the text for "have a nice day" so you can have different text every time you press the button.
